kclvm/api/src/service/jsonrpc.rs

```diff
@@ -91,6 +91,14 @@ fn register_kclvm_service(io: &mut IoHandler) {
         };
         futures::future::ready(catch!(kclvm_service_impl, args, get_schema_type))
     });
+    io.add_method("KclvmService.GetFullSchemaType", |params: Params| {
```

Comment:
Can you help add the missing API list? Including Test, RenameCode, RenameFile, etc?

Besides, add function list in the `register_builtin_service` function.

Reply:
fixed